src/test/java/com/puppycrawl/tools/checkstyle/api/ScopeTest.java

```diff
@@ -72,24 +71,15 @@ public void testMixedCaseSpaces() {
         assertEquals(Scope.ANONINNER, Scope.getInstance("AnonInner"), "Invalid scope");
     }
 
+    @DefaultLocale("tr_TR")
     @Test
-    public void testMixedCaseSpacesWithDifferentLocales() {
-        final Locale[] differentLocales = {new Locale("TR", "tr") };
```

Comment:
TR and tr was replaced with tr_TR. Will this test both locales like before?

Reply:
There was one item in the array: `Locale(country=TR, language=tr`, or just `tr_TR` in [BCP 47](ftp://ftp.isi.edu/in-notes/bcp/bcp47.txt) notation.